public/lessons/runtime-analysis/problem-solving.mdx

```diff
@@ -1,15 +1,14 @@
 ## Planning a Solution
 
-When looking at a new problem on $O(fast)$ or in a programming contest, take a careful look at the time limit and the input specification to determine what time complexity will be necessary to solve it.
+When looking at a new problem on O(fast) or in a programming contest, take a careful look at the time limit and the input specification to determine what time complexity will be necessary to solve it.
 
 A helpful rule-of-thumb in competitive programming is that computers can perform around $10^8$ operations in a second.
 You'll gain a better intuition for how much you can push this limit as you solve more problems.
 
 On O(fast), problems can be submitted in C++, Java, and Python. Of the three languages, C++ runs the fastest.
 Java runs a bit slower than C++, and Python runs a bit slower than Java. Most problems on O(fast) will pass in all 3 languages as
-long as the Big-O of the solution is small enough. However, programming contest on websites like Codeforces and Kattis won't be as forgiving.
-It's important to consider the performance of a programming language when deciding to learn it. 
-
+long as the Big-O of the solution is small enough. However, collegiate programming contests like the ICPC won't be as forgiving.
```

Comment:
Maybe it's just me, but "the Big-O of the solution" sounds somehow... improper. 
Do you think we can instead say something like "the time complexity of the solution is fast enough"?